_rules/SC4-1-2-button-has-name.md

```diff
@@ -19,7 +19,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any element with the role of `button`, except for `input` elements of `type="image"` that are [visible on the page][] or [exposed to assistive technologies][].
+The rule applies to any element with the [semantic role](#semantic-role) of `button`, except for `input` elements of `type="image"` that are [visible on the page][] or [exposed to assistive technologies][].
```

Comment:
This is ambiguous. Does the exception only apply to element visible and exposed to assistive technologies, or does it refer to all buttons?